DisplayInfo/CMakeLists.txt

```diff
@@ -67,16 +67,25 @@ if (PLUGIN_DISPLAYINFO_BCM_VERSION_MAJOR)
 endif()
 
 if (USE_DEVICESETTINGS)
-    find_package(DS REQUIRED)
-    find_package(IARMBus REQUIRED)
+    find_package(DS)
+    find_package(IARMBus)
     target_include_directories(${MODULE_NAME} PRIVATE ${IARMBUS_INCLUDE_DIRS}
         ${DS_INCLUDE_DIRS}
         ../helpers)
     target_link_libraries(${MODULE_NAME} PRIVATE
         ${NAMESPACE}Plugins::${NAMESPACE}Plugins
         ${IARMBUS_LIBRARIES}
-        ${DS_LIBRARIES}
-	displayinfo-soc)
+        ${DS_LIBRARIES})
+
```

Comment:
These changes are going to effect the prod builds. In the previous code displayinfo-soc is mandatory and build will fail, if it doesn't find this lib.
In this change build will continue.
I suggest to keep the original code as it is and add a new elseif for RDK_SERVICES_L1_TEST, DRDK_SERVICES_L2_TEST, where these libs can be ignored.